lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -335,6 +342,30 @@ export class SearchFacetFiltersService implements OnDestroy {
         };
     }
 
+    private loadCategoryNames(bucketList: FacetFieldBucket[]) {
+        bucketList.forEach((item) => {
+            const categoryId = item.label.split('/').pop();
+            this.categoryService.getCategory(categoryId)
+                .pipe(
+                    concatMap((categoryEntry) => this.categoryService.searchCategories(categoryEntry.entry.name)),
+                    catchError(error => {
+                        return throwError(error);
+                    }),
+                )
+                .subscribe(
+                    result => {
+                        const nextAfterGeneralPathPartIndex = 3;
+                        const pathSeparator = '/';
+                        const currentCat = result.list.entries.filter(entry => entry.entry.id === categoryId)[0];
+                        const path = currentCat.entry.path.name.split(pathSeparator).slice(nextAfterGeneralPathPartIndex).join('/');
+
+                        item.display = path ? `${path}/${currentCat.entry.name}` : currentCat.entry.name;
+                    },
+                    error => console.log(`Error: ${error}`)
```

Comment:
should we log it to console or should we rather let user know that something is not loaded correctly but displaying Snackbar? What do you think? @DenysVuika @MichalKinas @nikita-web-ua